tests/e2e/hcp_machine_pool_test.go

```diff
@@ -762,6 +762,78 @@ var _ = Describe("HCP MachinePool", ci.Day2, ci.FeatureMachinepool, func() {
 
 		})
 
+	It("can create machinepool with disk size - [id:69144]", ci.High, func() {
+		By("Create additional machinepool with disk size specified")
+		replicas := 1
+		machineType := "r5.xlarge"
+		subnetId := vpcOutput.PrivateSubnets[0]
+		name := helper.GenerateRandomName("np-69144", 2)
+		diskSize := 249
+		mpArgs := &exec.MachinePoolArgs{
+			Cluster:            helper.StringPointer(clusterID),
+			AutoscalingEnabled: helper.BoolPointer(false),
+			Replicas:           helper.IntPointer(replicas),
+			SubnetID:           helper.StringPointer(subnetId),
+			MachineType:        helper.StringPointer(machineType),
+			AutoRepair:         helper.BoolPointer(true),
+			Name:               helper.StringPointer(name),
+			DiskSize:           helper.IntPointer(diskSize),
+		}
+
+		_, err := mpService.Apply(mpArgs)
+		Expect(err).ToNot(HaveOccurred())
+		defer func() {
+			_, err = mpService.Destroy()
+			Expect(err).ToNot(HaveOccurred())
+		}()
+
+		By("Verify the parameters of the created machinepool")
+		mpResponseBody, err := cms.RetrieveClusterNodePool(cms.RHCSConnection, clusterID, name)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(mpResponseBody.AWSNodePool().RootVolume().Size()).To(Equal(diskSize))
+		Expect(mpResponseBody.AWSNodePool().InstanceType()).To(Equal(machineType))
+
+		By("Update disksize is not allowed ")
```

Comment:
Negative checkpoints should be added into different test case with the medium priority

Reply:
Add this step to case id 76345

Reply:
To avoid code duplication, it's written here since it needs to depend on a successfully created machine pool. WDYT? @aaraj7

Reply:
I will prefer to add it with id 76345 as negative checkpoints should have medium level priority cc: @radtriste Could you please confirm with the same?

Reply:
@aaraj7 I've moved it to `76345`.